tests/bgm.rs

```diff
@@ -0,0 +1,12 @@
+use bms_rs::{lex::parse, parse::{rng::RngMock, Bms}};
+
+#[test]
+fn bgm_obj_test() {
+    let src = std::fs::read_to_string("tests/nc_mx.bme").unwrap();
+    let ts = parse(&src).unwrap();
+    let rng = RngMock([1]);
+    let bms = Bms::from_token_stream(&ts, rng).unwrap();
+    for b in bms.notes.bgms() {
+        println!("{:?}", b);
+    }
+}
```

Comment:
I think that this test is doing same thing as `test_nc` in `tests/files.rs`, so it should not be added.